.github/workflows/release.yml

```diff
@@ -47,11 +47,11 @@ jobs:
         uses: google-github-actions/setup-gcloud@v0
       - name: Release notes
         run: |
-          go run ./cmd/release-notes/main.go -version "${GITHUB_REF_NAME}" > ${{ runner.temp }}/releasenotes
+          go run ./cmd/release-notes/main.go -version "${GITHUB_REF_NAME}" -owner "${GITHUB_ACTOR}" > ${{ runner.temp }}/releasenotes
```

Comment:
@sourishkrout I added a new secret... I can revert this but it was basically the only way I could get pass this so it makes the gorelease more flexible for sure.